cassandra-dac/src/main/java/org/sunbird/helper/CassandraConnectionManager.java

```diff
@@ -1,189 +1,13 @@
-					msg = String.format("Datacenter: %s; Host: %s; Rack: %s", host.getDatacenter(), host.getAddress(),
-							host.getRack());
-					ProjectLogger.log(msg);
-				}
-			}
-			   }catch(Exception e){
-			       ProjectLogger.log("Error occured while creating connection :",e);
-				   throw new ProjectCommonException(ResponseCode.internalError.getErrorCode(), e.getMessage(), ResponseCode.SERVER_ERROR.getResponseCode());
-			   }
-		        
-		        if(null != cassandraSessionMap.get(keyspace)){
-		        	connection = true;
-		        }
-		        return connection;
-	}
-	
-	/**
-	 * This method provide session for given keyspace
-	 * @param  keyspaceName String
-	 * @return Session Session
-	 */
-	public static Session getSession(String keyspaceName) {
-		if(null == cassandraSessionMap.get(keyspaceName)){
-			throw new ProjectCommonException(ResponseCode.internalError.getErrorCode(), Constants.SESSION_IS_NULL+keyspaceName, ResponseCode.SERVER_ERROR.getResponseCode());
-		}
-        return cassandraSessionMap.get(keyspaceName);
-    }
- 
-   
-	/**
-	 * 
-	 * @param ip String
-	 * @param port String
-	 * @param userName String
-	 * @param password String
-	 * @param poolingOptions PoolingOptions
-	 * @return Cluster Cluster
-	 */
-	private static Cluster createCluster(String ip,String port,String userName, String password,PoolingOptions poolingOptions){
-		return Cluster
-				.builder()
-				.addContactPoint(ip)
-				.withPort(Integer.parseInt(port))
-				.withProtocolVersion(ProtocolVersion.V3)
-				.withRetryPolicy(DefaultRetryPolicy.INSTANCE)
-				.withTimestampGenerator(new AtomicMonotonicTimestampGenerator())
-				.withPoolingOptions(poolingOptions).withCredentials(userName,password).build();	
-	}
-	
-	/**
-	 * 
-	 * @param ip String
-	 * @param port String
-	 * @param poolingOptions PoolingOptions
-	 * @return Cluster Cluster
-	 */
-	private static Cluster createCluster(String ip,String port, PoolingOptions poolingOptions){
-		return Cluster
-		.builder()
-		.addContactPoint(ip)
-		.withPort(Integer.parseInt(port))
-		.withProtocolVersion(ProtocolVersion.V3)
-		.withRetryPolicy(DefaultRetryPolicy.INSTANCE)
-		.withTimestampGenerator(new AtomicMonotonicTimestampGenerator())
-		.withPoolingOptions(poolingOptions).build();
-	}
-	
-	/**
-	 * This class will be called by registerShutDownHook to 
-	 * register the call inside jvm , when jvm terminate it will call
-	 * the run method to clean up the resource.
-	 */
-	static class ResourceCleanUp extends Thread {
-	      @Override
-		  public void run() {
-			  ProjectLogger.log("started resource cleanup Cassandra.");
-			  for(Map.Entry<String,Session> entry: cassandraSessionMap.entrySet()){
-					cassandraSessionMap.get(entry.getKey()).close();
-					cassandraclusterMap.get(entry.getKey()).close();
-				}
-			  ProjectLogger.log("completed resource cleanup Cassandra.");
-		  }
-	}
+  boolean createConnection(String ip,String port,String userName,String password,String keyspace);
```

Comment:
Write the documentation for each class label and  method label which will clearly indicate what the purpose of class and purpose of method , it's argument and return type.